### PR TITLE
perf(ZenMarket): make it more readable & improve code quality

### PR DESCRIPTION
**Pull request by darkvillager2**
## Description 

make it more readable & improve code quality
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/Z/ZenMarket/presence.ts [hunk 1]
```diff
@@ -17,249 +17,256 @@ presence.on("UpdateData", async () => {
...
+								label: "View On ZenMarket",
+								url: href,
+							},
+							{
+								label: "View On Other Market",
+								url: item?.productPage,
+							},
+						];
+					}
+				}
```
Comment (darkvillager2) [inline websites/Z/ZenMarket/presence.ts:248]: Would love to shorten this with a dynamic button name ;)

Comment (Bas950) [reply websites/Z/ZenMarket/presence.ts:247]: If @theusaf agrees you can IMO.

**Comment by Bas950** [comment]
@darkvillager2 Rebase is needed.

File: websites/Z/ZenMarket/presence.ts (hunk 1 above)
Comment (theusaf) [reply websites/Z/ZenMarket/presence.ts:247]: I think that's fine.

**Review by Bas950 — APPROVED** [review]

**Review by theusaf — APPROVED** [review]

**Pull request merged**